pkg/cmd/types.go

```diff
@@ -29,6 +29,15 @@ type Service struct {
 	Writable     bool                           `json:"writable"`
 }
 
+type ServiceClass struct {
```

Comment:
would it make more sense to call this ExternalServiceMetaData or something like that?